pkg/load/load.go

```diff
@@ -32,6 +47,48 @@ func Config(path string) (*api.ReleaseBuildConfiguration, error) {
 	if err := yaml.Unmarshal([]byte(raw), configSpec); err != nil {
 		return nil, fmt.Errorf("invalid configuration: %v\nvalue:\n%s", err, raw)
 	}
+
+	// If configresolver address info is provided, GET a copy of the config from there
+	if info != nil {
+		req, err := http.NewRequest("GET", fmt.Sprintf("http://%s/config", info.Address), nil)
```

Comment:
I don't think we want to prefix with `http://` here

Reply:
Without `http://`, error: parse 127.0.0.1:8080/config: first path segment in URL cannot contain colon

Reply:
I was thinking the user would pass it in with the flag -- what happens if we want to serve this between clusters and use a secure route? With this implementation we'd need to change the code, but if we pass `--config-resolver-url https://whatever.com` we would just change runtime parameters

Reply:
Yeah, didn't initially realize that. Fixing it now